mempool/collector.go

```diff
@@ -112,7 +118,7 @@ func (t *MempoolDataCollector) mempoolTxns() ([]exptypes.MempoolTx, error) {
 			VoutCount: len(msgTx.TxOut),
 			Vin:       exptypes.MsgTxMempoolInputs(msgTx),
 			Coinbase:  blockchain.IsCoinBaseTx(msgTx),
-			Hash:      hash,
+			Hash:      hash.String(),
```

Comment:
We already have `hashStr`.  Why make a chainhash.Hash in this function?

Reply:
reverted to hashStr